### PR TITLE
update privacy policy

### PR DESCRIPTION
**Pull request by steve-codurance**

### CODE REVIEW

File: policies/privacy.html [hunk 1]
```diff
@@ -22,58 +28,62 @@ <h4>Third party websites</h4>
...
+            <ul>
+                <li>Attending meetings and/or during telephone calls to us;</li>
+                <li>Corresponding with us by email, in which case we may retain the content of your email messages together with your email address and our responses;</li>
+                <li>Supplying us with your personal information by signing up to receive a newsletter or to attend an event;</li>
+                <li>Supplying us with your personal information as part of the recruitment process;</li>
+                <li>Using our website, during which other personal data may be automatically collected, such as your IP address (please see section on cookies, below).</li>
+            </ul>
+        </div>
+        <div class="margin-bottom-30">
+            <h4>How will Codurance us this information?</h4>
```
Comment (Gryff) [inline policies/privacy.html]: Typo - `us` -> `use`

File: policies/privacy.html [hunk 2]
```diff
@@ -22,58 +28,62 @@ <h4>Third party websites</h4>
...
-                <li>
-                    to enable us to review, develop and improve the products, services and special offers online.
-                </li>
-                <li>
-                    to occasionally carry out market research.
-                </li>
-                <li>
-                    to send you details of Codurance promotions, products, services, special offers and rewards that we think will be of interest to you. It's up to you to decide whether or not you want to receive this information.
-                </li>
+                <li>for "service administration purposes", which means that the Codurance may contact you for reasons related to the service you have signed up for (e.g. to provide you with password reminders or to notify you that a particular service has been suspended for maintenance);</li>
```
Comment (Gryff) [inline policies/privacy.html]: 'the' Codurance?

File: policies/privacy.html (hunk 1 above)
Comment (steve-codurance) [reply policies/privacy.html]: Thanks!

File: policies/privacy.html (hunk 2 above)
Comment (steve-codurance) [reply policies/privacy.html]: Thanks again!

**Pull request merged**